Program.cs

```diff
@@ -15,252 +15,273 @@ public static void printsContentsRefrigerator(List<Refrigerator> allfrige)
         {
             foreach (Refrigerator friger in allfrige)
             {
-                Console.WriteLine(friger.ToString()); 
+                Console.WriteLine(friger.ToString());
                 foreach (Shelf shelf in friger._shelfes)
                 {
                     Console.WriteLine(shelf.ToString());
                     foreach (Item item in shelf._itemsOnShelf)
                     {
-                        Console.WriteLine("  "+item.ToString()); 
+                        Console.WriteLine("  " + item.ToString());
                     }
                 }
 
             }
         }
-        public static Item.Kashrot checkKosherInput()
+        public static void CheckingIntegrityInput(int userInput)
```

Comment:
avoid static method in such an exercise like this.
the main purpose of static is to exposure the method to all layers of the code.